app/views/admin/users/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Registered Users</h1>
+
+  <% @users.each do |user| %>
+    <% if user.role == "member" %>
```

Comment:
you could also define a `member?` instance method on the model to make the readability of this a bit nicer.